src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,185 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.NoSuchElementException;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head = null;
+    private Node<T> tail = null;
+    private int size;
+
+    public Node<T> getHead() {
+        return head;
+    }
+
+    public Node<T> getTail() {
+        return tail;
+    }
```

Comment:
You don't need this getters.

Reply:
that's true, remove them and reference variables directly